test/action_view/component_test.rb

```diff
@@ -108,11 +108,20 @@ def test_renders_partial_template
   end
 
   def test_renders_content_for_template
-    result = render_inline(ContentForComponent)
+    result = render_inline(ContentForComponent, title: "Hi!", footer: "Bye!") { "Have a nice day."}
 
-    assert_html_matches "<div>Hello content for</div>", result.to_html
+    assert_html_matches "<div>\nHi!\nHave a nice day.\nBye!\n</div>", result.to_html
   end
 
+  # def test_renders_content_for_template_with_block
+  #   result = render_inline(ContentForComponent, footer: "Bye!") {|comp| "<% content_for(:title) { 'My Title'} %>Have a nice day."}
+  #
+  #   assert_includes result.text, "Hi!"
+  #   assert_includes result.text, "Have a nice day."
+  #   assert_includes result.text, "Bye!"
+  #   assert_html_matches "<div>\nMy Title\nHave a nice day.\nBye!\n</div>", result.to_html
+  # end
```

Comment:
I really want to be able to unit test the various options but I can't figure out the incantations of providing an ERB template. I assume there is a way to do it but I haven't gone spelunking through the Rails helper tests to figure it out yet.